pkg/tsdb/cloudwatch/live.go

```diff
@@ -107,7 +107,7 @@ func (r *logQueryRunner) publishResults(orgID int64, channelName string) error {
 
 // executeLiveLogQuery executes a CloudWatch Logs query with live updates over WebSocket.
 // A WebSocket channel is created, which goroutines send responses over.
-//nolint: staticcheck // plugins.DataResponse deprecated
+//nolint: staticcheck // legacydata.DataResponse deprecated
```

Comment:
`//nolint: staticcheck // legacydata.DataResponse deprecated` aren't relevant in this file anymore, right?